angular/package.json

```diff
@@ -40,7 +40,7 @@
     "angular-translate": "~2.7.2",
     "angular-translate-loader-url": "~2.7.2",
     "angular-ui-bootstrap": "~> 0.14.3",
-    "angular_record_store": "~2.6.0",
+    "angular_record_store": "2.6.1",
```

Comment:
I think you can leave this at `~2.6.0`, which allows patch level changes (so, 'give me the latest up to but not including 2.7.0')

Then we won't have to change it for 2.6.2, etc.